app/services/global_search/common.rb

```diff
@@ -0,0 +1,23 @@
+module GlobalSearch
+
+  module Common
+
+    extend ActiveSupport::Concern
+
+    included do
+      attr_reader :facility, :query, :user
+    end
+
+    def initialize(user = nil, facility = nil, query = nil)
```

Comment:
Good call on inheritance. I've been in a concerns mindset with most of the recent work and didn't give that possibility a lot of consideration.